drivers/web-worker-driver/build.gradle

```diff
@@ -5,39 +5,43 @@ plugins {
 }
 
 kotlin {
-  js {
-    browser {
+  [js(), wasmJs()].forEach {
+    it.browser {
       testTask {
         useKarma {
           useChromeHeadless()
         }
       }
     }
   }
+  applyDefaultHierarchyTemplate {
+    it.common {
+      it.group("jsCommon") {
```

Comment:
Is there any need for this to be `jsCommon` instead of leaving it as the default `common`?

Reply:
Yeah, you are right, my bad, I'll remove this group.